### DISCUSSION TITLE
How to change the default new project setting?

### DISCUSSION THREAD

**Discussion by DJDQfff** [stride3d/stride#1431] [Q&A] [answered]
When i creat a new project ,the default project seems to be fixed.
Now i want to change the default setting.
For example ,for UWP paltform,the default platform is 16299, the windowsSDK i installed in 19041.Each time i creat a new uwp platform,i would get a fail.In fact ,i  can rewrite the .csproj file content,and then the project can work correctly.

**Comment by tebjan** [comment, marked as answer]
A Stride game is just a visual studio solution, so you can change everything there like in a normal C# project. If you want to change platforms, check the manual here: https://doc.stride3d.net/latest/en/manual/platforms/add-or-remove-a-platform.html
